test/e2e/models/CodeServer.ts

```diff
@@ -5,6 +5,7 @@ import { CODE_SERVER_ADDRESS } from "../../utils/constants"
 // See Playwright docs: https://playwright.dev/docs/pom/
 export class CodeServer {
   page: Page
+  editorSelector = "div.monaco-workbench"
```

Comment:
🛠️ I realized how many places I was using this in the model so refactored it into a property of the class